tests/unit/misc/test_utilcmds.py

```diff
@@ -0,0 +1,152 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2015-2016 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for qutebrowser.misc.utilcmds."""
+
+import contextlib
+import logging
+import os
+import pytest
+import signal
+import time
+
+_hunter_available = False
+try:
+    import hunter  # pylint: disable=unused-import
+    _hunter_available = True
+except ImportError:
+    pass
+
+from qutebrowser.misc import utilcmds as utilcmdsmod
```

Comment:
I used `test_lineparser` as a "do like the romans do" scratchpad and thought that was to indicate the module under test. Coming again, that was probably just so the fixture can have a nice name.